tmp/a/BUILD

```diff
@@ -0,0 +1,4 @@
+python_library(
```

Comment:
Please move test related files to a testproject with the proper name. Or you can create them on the fly in the tests like https://github.com/pantsbuild/pants/blob/44c1f887b6f63d87fb11999cb9db9582409c6e12/tests/python/pants_test/backend/jvm/tasks/test_jvm_run.py#L31